sigle/src/modules/editor/ActiveNode.tsx

```diff
@@ -0,0 +1,56 @@
+import {
+  BulletedListLight,
+  CodeLight,
+  Heading2Light,
+  Heading3Light,
+  NumberedListLight,
+  QuoteLight,
+} from '../../icons';
+import { Editor } from '@tiptap/react';
+import { TextIcon } from '@radix-ui/react-icons';
+
+export const activeNode = (editor: Editor) => {
+  if (editor.isActive('heading', { level: 2 })) {
+    return {
```

Comment:
return properties from command list rather than hard-code

Reply:
@pradel regarding this one, is the idea to filter the command list and look for a match with, say the title, which would ofc require manually passing it for each node, or do you know if there is a more programmatic way to do it?